.github/workflows/security-scan.yml

```diff
@@ -17,12 +18,12 @@ jobs:
         uses: actions/checkout@v4
       - name: Run Trivy in table mode
         # Table output is only useful when running on a pull request or push.
-        if: contains(fromJSON('["pull_request"]'), github.event_name)
+        if: contains(fromJSON('["push"]'), github.event_name)
         uses: aquasecurity/trivy-action@0.28.0
```

Comment:
latest trivy-action is 0.29.0

Reply:
`dependabot` will let us know if this needs to be updated.